topics/sequence-analysis/tutorials/annotation-with-prokka/tutorial.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: tutorial_hands_on
+topic_name: sequence-analysis
+tutorial_name: "annotation-with-prokka"
+---
+
+# Introduction
+{:.no_toc}
+
+In this section we will use a software tool called Prokka to annotate the draft genome sequence produced in the previous [tutorial](/modules/spades/index.md). Prokka is a “wrapper”; it collects together several pieces of software (from various authors), and so avoids “re-inventing the wheel”.
```

Comment:
The link to Spades tutorial will not work there. Do you want to link to the one online?

Reply:
Thanks - removed link and mention.